src/smt/dump.cpp

```diff
@@ -42,6 +44,16 @@ CVC4dumpstream& CVC4dumpstream::operator<<(const NodeCommand& nc)
   return *this;
 }
 
+#else
+
+CVC4dumpstream& CVC4dumpstream::operator<<(const Command& c) { return *this; }
+CVC4dumpstream& CVC4dumpstream::operator<<(const NodeCommand& nc)
+{
+  return *this;
+}
+
+#endif
```

Comment:
Nit: Since preprocessing directives are not indented, it gets hard to track which `#endif` belongs to which `#if`. So, I find it helpful to mention the relevant condition next to `#endif`.
```suggestion
#endif /* CVC4_DUMPING && !CVC4_MUZZLE */
```